ecosystem/typescript/sdk/src/tests/e2e/aptos_client.test.ts

```diff
@@ -48,6 +48,14 @@ test("gets genesis resources", async () => {
   expect(accountResource).toBeDefined();
 });
 
+// test("gets object", async () => {
```

Comment:
yea.. you would need to create an object and then query it to test it. can do something like that
```
const alice = new AptosAccount();
const faucetClient = getFaucetClient();
await faucetClient.fundAccount(alice.address(), 100000000);

const provider = new Provider({
        fullnodeUrl: NODE_URL,
        indexerUrl: NODE_URL,
      });
const aptosToken = new AptosToken(provider)

const txn = await client.waitForTransactionWithResult(
        await aptosToken.createCollection(alice, "Alice's simple collection","AliceCollection", "https://aptos.dev", 5),
        { checkSuccess: true },
      );
const objectAddress = txn.changes[0].address // should be the new object address
```

Reply:
Added